### DISCUSSION TITLE
Css Classes toggle

### DISCUSSION THREAD

**Discussion by Mte90** [zauberzeug/nicegui#4013] [Ideas / Feature Requests]
Like we have in jQuery so a code like this `.classes(remove='hidden', add='')` can be `.classes(toggle="hidden")`.

**Comment by falkoschindler** [comment]
Great idea, @Mte90!
I created PR #4019 to implement this feature.

**Reply by falkoschindler** [reply]
This feature has been released in version 2.7.0. ✔️

**Comment by Mte90** [comment]
thank you so much :-D
I think that this project can copy a lot from jQuery but I just started using it.
